releasenotes/notes/profiling-export-tags-fixed-9d383890c8e5c4c1.yaml

```
@@ -0,0 +1,5 @@
+---
+upgrade:
+  - |
+    The profiler will not load tags from the DD_TAGS environment variable only
+    once at start.
```

```suggestion
    The profiler will only load tags from the DD_TAGS environment variable
    once at start.
```

Or else, probably missing something like:

```
The profiler will not load tags from the DD_TAGS environment variable on every flush, now only once at start.
```

or something.